spec/models/endpoint_spec.rb

```diff
@@ -93,11 +86,12 @@
 
   describe '.default_storage_root_endpoint_type' do
     it 'returns the default endpoint type object for the storage root' do
-      EndpointType.seed_from_config
+      # db already seeded
       expect(Endpoint.default_storage_root_endpoint_type).to be_a_kind_of EndpointType
     end
 
     it "raises RecordNotFound if the default endpoint type doesn't exist in the db" do
+      skip('database seeded before running tests; not super-trivial to destroy relevant objects')
       expect { Endpoint.default_storage_root_endpoint_type }.to raise_error(ActiveRecord::RecordNotFound)
     end
```

Comment:
If we're not going to use this test, why not just delete it.

Reply:
because @jmartin-sul intends to fix it and he also didn't want to lose the test.